templates/mod/ssl.conf.erb

```diff
@@ -10,6 +10,9 @@
   SSLPassPhraseDialog  builtin
   SSLSessionCache "shmcb:<%= @session_cache %>"
   SSLSessionCacheTimeout 300
+<% if scope.function_versioncmp([@apache_version, '2.4']) >= 0 -%>
+  SSLStaplingCache "shmcb:<%= @stapling_cache %>"
```

Comment:
I may be wrong, but shouldn't the `SSLStaplingCache` only be set if ssl_stapling is activated?

I propose this conditional:

```ruby
if @ssl_stapling && scope.function_versioncmp([@apache_version, '2.4']) >= 0
```